### DISCUSSION TITLE
Problem logging in

### DISCUSSION THREAD

**Discussion by Muerio** [DialmasterOrg/Youtarr#24] [General]
Hi,

so I wanted to give Youtarr a try and move away from my shell scripts.
I got the container running, opened the interface over port 3087 and was greeted by the login screen.
Even though I logged in using plex, I'm only see the login screen and can't change to the configuration.

Any ideas what could be the problem here?

**Comment by dialmaster** [comment]
Can you give me a little more information:
What kind of system are you running on (Windows/Linux).
What were the steps you followed to login?
Did you get any errors during login?

Also, if you are willing to take a look in your <youtarr_directory>/config/config.json, did the plexApiKey get populated with a value?

If you can try logging in again and give me the output of your browser console log and also the youtarr docker container log when you try I can probably figure out the issue and fix it :)

Thanks for helping me test and make this better!

**Reply by Muerio** [reply]
OS: Ubuntu Server 22.04.02 LTS

Steps to login:
1. Clone repo and checkout version 1.3.3
2. Run setup.sh => Checked config.json, "youtubeOutputDirectory" was set correctly
3. Run start.sh => Downloads container and runs it without problems on Docker version 20.10.21 (build 20.10.21-0ubuntu1~22.04.3)
4. Open web interface in Edge/Chrome/Firefox on another machine using <local server ip>:3087
5. Click "LOGIN VIA PLEX" => New tab opens => Plex login completed without errors

config.json does not contain an API key for plex

The console log for the youtarr page shows the following error when logging in:
![image](https://github.com/dialmaster/Youtarr/assets/11858860/19014bdb-d181-4b20-b5a4-8a554ac0c425)


The Plex login page itself doesn't show any errors in the logs.

The docker container logs report the following:
_docker logs youtarr --details_
 Looking for next job to start
 Scheduling task to run at: 0 * * * *
 Auto-downloads disabled
 Youtube downloads directory from config: /usr/src/app/data
 Server listening on port 3011

Let me know if you need further information & thank you for the support :)

**Reply by dialmaster** [reply]
Thanks for all the info. I have to fit this in around my day job, but I'll try to setup a system to replicate your config, see if I can reproduce your issue, and fix it. I'll ping once I have a fix (or if I need more details).

I opened an issue to look into this and will make it my highest priority: https://github.com/dialmaster/Youtarr/issues/25

**Reply by dialmaster** [reply]
@Muerio One more followup question just to ensure I setup a mirror system to duplicate your issue properly:
You have Plex running on the same Ubuntu machine as you are trying to run Youtarr, correct?
I assume that you can directly access plex on the host machine using the standard port, eg: localhost:32400 ?
Are you running Plex in it's own Docker container, or just directly in the host?

**Reply by Muerio** [reply]
Plex is running directly on the host. Accessing Plex using localhost works.

**Reply by dialmaster** [reply]
Hi @Muerio I have a fix ready and merged to the main branch in my github repo.
I have tested it on an Ubuntu system running the same version as you, with a Plex server running directly on the host Ubuntu.
Please do a 'git pull' and then re-run setup.sh.
Once that is done stop and restart your running docker container for youtarr:
./stop.sh
./start.sh

Let me know if that resolves things for you.
I'll keep the issue open until I get confirmation that it is fixed.

**Reply by Muerio** [reply]
Hi @dialmaster,

great, it works!
Thank you for the quick response.
